lib/components/DraftailEditor.test.js

```diff
@@ -389,6 +389,34 @@ describe("DraftailEditor", () => {
     expect(wrapper.state("hasFocus")).toBe(false);
   });
 
+  it("#onFocus", () => {
+    const onFocus = jest.fn();
+    const wrapper = shallowNoLifecycle(<DraftailEditor onFocus={onFocus} />);
+
+    wrapper.instance().onFocus();
+
+    expect(onFocus).toHaveBeenCalled();
+
+    shallowNoLifecycle(<DraftailEditor />)
+      .instance()
+      .saveState();
+  });
+
+  it("#onBlur", () => {
+    const onBlur = jest.fn();
+    const onSave = jest.fn();
+    const wrapper = shallowNoLifecycle(<DraftailEditor onBlur={onBlur} onSave={onSave} />);
```

Comment:
I think CI fails because this line is longer than the formatting allowed by Prettier.